pyproject.toml

```diff
@@ -58,6 +58,9 @@ xarray = ">=22.6.0"
 tabulate = ">=0.8.10"
 
 
+[tool.poetry.group.dev.dependencies]
+SQLAlchemy = "^1.4.41"
```

Comment:
Only question is why put this in a separate group versus in the list above?

Otherwise looks good.

Reply:
Thank you for your comment. I didnt intended it, whereas the group created just by `poetry add --dev sqlalchemy`.

I'll it.